linux_pipeline/Jenkinsfile_azure_lis_hotfix_functional

```diff
@@ -0,0 +1,344 @@
+	Command += " -XMLSecretFile ${Azure_Secrets_TESTONLY_File}"
+	Command += " -TestPlatform 'Azure'"
+	Command += " -TestLocation 'northeurope'"
+	Command += " -RGIdentifier 'CAPTURE${BUILD_NUMBER}'"
+	Command += " -ResourceCleanup Delete"
+	Command += " -TestIterations 1"
+	Command += " -ExitWithZero"
+	Command += " -StorageAccount 'ExistingStorage_Standard'"
+	Command += " -TestNames 'CAPTURE-VHD-BEFORE-TEST'"								
+	Command += " -OverrideVMSize 'Standard_DS2_v2'"
+	Command += " -ARMImageName '${currentVHD}'"
+	
+	//Get the Custom Kernel
+	if ( CUSTOM_KERNEL != "" ) {
+		Command += " -CustomKernel '${CUSTOM_KERNEL}'"
+	}
+	
+	Command += " -CustomParameters 'DiskType=Unmanaged'"
+	println Command
+	RunPowershellCommand(Command)
+	junit "Report\\*-junit.xml"
+	archiveArtifacts '*-TestLogs.zip'
+	def CapturedVHD = readFile 'CapturedVHD.azure.env'
+	def StashName = currentVHD.replace(" ","_")
+	writeFile file: "BASE-${StashName}-CapturedVHD.azure.env", text: "${CapturedVHD}"
+	stash includes: "BASE-${StashName}-CapturedVHD.azure.env", name: "BASE-${StashName}-CapturedVHD.azure.env"
+	println("Captured VHD : ${CapturedVHD}")											
+}
+
+def captureVHD_FunctionalTest(currentVHD) {
+	Prepare()
+	println "CAPTURE-VHD-BEFORE-TEST"								
+	def Command = ".\\Run-LisaV2.ps1"
+	Command += " -XMLSecretFile ${Azure_Secrets_TESTONLY_File}"
+	Command += " -TestPlatform 'Azure'"
+	Command += " -TestLocation 'northeurope'"
+	Command += " -RGIdentifier 'CAPTURE${BUILD_NUMBER}'"
+	Command += " -ResourceCleanup Delete"
+	Command += " -TestIterations 1"
+	Command += " -ExitWithZero"
+	Command += " -StorageAccount 'ExistingStorage_Standard'"
+	Command += " -TestNames 'CAPTURE-VHD-BEFORE-TEST'"								
+	Command += " -OverrideVMSize 'Standard_DS2_v2'"
+	Command += " -ARMImageName '${currentVHD}'"
+	
+	//Get the Custom Kernel
+	if ( CUSTOM_KERNEL != "" ) {
+		Command += " -CustomKernel '${CUSTOM_KERNEL}'"
+	}
+	//Get the Custom LIS
+	Command += " -CustomLIS '${rpmURL}'"
+	println "Set customLIS as ${rpmURL}"
+											
+	Command += " -CustomParameters 'DiskType=Unmanaged'"
+	println Command
+	RunPowershellCommand(Command)
+	junit "Report\\*-junit.xml"
+	archiveArtifacts '*-TestLogs.zip'
+	def CapturedVHD = readFile 'CapturedVHD.azure.env'
+	def StashName = currentVHD.replace(" ","_")
+	writeFile file: "TEST-${StashName}-CapturedVHD.azure.env", text: "${CapturedVHD}"
+	stash includes: "TEST-${StashName}-CapturedVHD.azure.env", name: "TEST-${StashName}-CapturedVHD.azure.env"
+	println("Captured VHD : ${CapturedVHD}")											
+
+}
+
+stage ("Prerequisite") {
+    node ("meta_slave") {
+        cleanWs()
+        git branch: env.GIT_BRANCH, url: env.GIT_REPO
+        stash includes: '**', name: 'LISAv2'
+        cleanWs()
+    }
+}
+
+currentStage = "Capture Image for Deploy Test"
+stage ("${currentStage}") {
+	def parellel_jobs = [:]
+	def delay = 0
+	println "DistroList ${DistroList}"
+	for (i = 0; i < total_test_images; i++)	{
```

Comment:
Please check out indentation rule of this file.

Reply:
fixed indentation rule